packages/data-point/README.md

```diff
@@ -1482,7 +1496,7 @@ To prevent unexpected results, **Hash** can only process **Plain Objects**, whic
 Hash entities expose a set of reducers: [mapKeys](#hash-mapKeys), [omitKeys](#hash-omitKeys), [pickKeys](#hash-pickKeys), [addKeys](#hash-addKeys), [addValues](#hash-addValues). You may apply one or more of these available reducers to a Hash entity. Keep in mind that those reducers will always be executed in a specific order:
 
 ```js
-mapKeys -> omitKeys -> pickKeys -> addKeys -> addValues
+omitKeys -> pickKeys -> mapKeys -> addValues -> addKeys
```

Comment:
These transforms are not executed in the order that's listed in the current readme.

Reply:
👍